internal/cmd/stack/local_preview.go

```diff
@@ -22,13 +22,25 @@ func localPreview() cli.ActionFunc {
 		}
 		ctx := context.Background()
 
+		var packagePath *string = nil
+		if cliCtx.Bool(flagProjectRootOnly.Name) {
+			root, err := getGitRepositorySubdir()
+			if err != nil {
+				return fmt.Errorf("couldn't get the packagePath: %w", err)
+			}
+			packagePath = &root
+		}
 		if !cliCtx.Bool(flagNoFindRepositoryRoot.Name) {
 			if err := internal.MoveToRepositoryRoot(); err != nil {
 				return fmt.Errorf("couldn't move to repository root: %w", err)
 			}
 		}
 
-		fmt.Println("Packing local workspace...")
+		if packagePath == nil {
+			fmt.Println("Packing local workspace...")
+		} else {
+			fmt.Printf("Packing '%s' as local workspace\n", *packagePath)
```

Comment:
A bit inconsistent now, with one being Println and trailing dots, while the other is Printf with an explicit newline and no dots 😅